oshmem/mca/spml/ucx/spml_ucx_component.c

```diff
@@ -293,6 +293,11 @@ static int spml_ucx_init(void)
     params.field_mask       |= UCP_PARAM_FIELD_ESTIMATED_NUM_PPN;
 #endif
 
+#if HAVE_DECL_UCP_PARAM_FIELD_NODE_LOCAL_ID
+    params.node_local_id = opal_process_info.my_local_rank;
+    params.field_mask   |= UCP_PARAM_FIELD_NODE_LOCAL_ID;
+#endif
```

Comment:
consider moving some initializations to common code (opal/mca/common/ucx)

Reply:
i'd suggest another pr for that, because we have many common things, not only this new param